### ISSUE TITLE
Ubuntu 14.04 compile error: undefined reference to `avformat_alloc_output_context2'

### ISSUE THREAD

**Issue by ckorn** [slowmoVideo/slowmoVideo#68]
The build of version 0.4 fails on Ubuntu 14.04:
```
/usr/bin/x86_64-linux-gnu-g++    -Wall -O2 -O3 -DNDEBUG    CMakeFiles/slowmoUI.dir/main.cpp.o CMakeFiles/slowmoUI.dir/mainwindow.cpp.o CMakeFiles/slowmoUI.dir/canvas.cpp.o CMakeFiles/slowmoUI.dir/canvasTools.cpp.o CMakeFiles/slowmoUI.dir/frameMonitor.cpp.o CMakeFiles/slowmoUI.dir/renderPreview.cpp.o CMakeFiles/slowmoUI.dir/dialogues/newProjectDialog.cpp.o CMakeFiles/slowmoUI.dir/dialogues/preferencesDialog.cpp.o CMakeFiles/slowmoUI.dir/dialogues/projectPreferencesDialog.cpp.o CMakeFiles/slowmoUI.dir/dialogues/progressDialog.cpp.o CMakeFiles/slowmoUI.dir/dialogues/renderingDialog.cpp.o CMakeFiles/slowmoUI.dir/dialogues/shutterFunctionDialog.cpp.o CMakeFiles/slowmoUI.dir/dialogues/shutterFunctionFrame.cpp.o CMakeFiles/slowmoUI.dir/dialogues/flowExaminer.cpp.o CMakeFiles/slowmoUI.dir/dialogues/tagAddDialog.cpp.o CMakeFiles/slowmoUI.dir/dialogues/aboutDialog.cpp.o CMakeFiles/slowmoUI.dir/moc_mainwindow.cxx.o CMakeFiles/slowmoUI.dir/moc_canvas.cxx.o CMakeFiles/slowmoUI.dir/moc_frameMonitor.cxx.o CMakeFiles/slowmoUI.dir/moc_renderPreview.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_newProjectDialog.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_preferencesDialog.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_projectPreferencesDialog.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_progressDialog.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_renderingDialog.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_shutterFunctionDialog.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_shutterFunctionFrame.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_flowExaminer.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_tagAddDialog.cxx.o CMakeFiles/slowmoUI.dir/dialogues/moc_aboutDialog.cxx.o CMakeFiles/slowmoUI.dir/qrc_resources.cxx.o  -o slowmoUI -rdynamic ../project/libsVproj.a ../lib/libsVvis.a ../libgui/libsVgui.a -lQtScript -lQtGui -lQtTest -lQtXml -lQtCore /usr/lib/x86_64-linux-gnu/libopencv_videostab.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_video.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_ts.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_superres.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_stitching.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_photo.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_ocl.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_objdetect.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_ml.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_legacy.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_imgproc.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_highgui.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_gpu.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_flann.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_features2d.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_core.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_contrib.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_calib3d.so.2.4.8 -lavformat -lavcodec -lavutil -lswscale ../lib/libsV.a ../lib/libsVinfo.a ../lib/libsVencode.a ../lib/libsVflow.a -lQtScript -lQtGui -lQtTest -lQtXml -lQtCore /usr/lib/x86_64-linux-gnu/libopencv_photo.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_legacy.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_video.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_objdetect.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_ml.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_calib3d.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_features2d.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_highgui.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_imgproc.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_flann.so.2.4.8 /usr/lib/x86_64-linux-gnu/libopencv_core.so.2.4.8 -lavformat -lavcodec -lavutil -lswscale 
../lib/libsVencode.a(ffmpegEncode_sV.c.o): In function `prepareDefault':
ffmpegEncode_sV.c:(.text+0x26f): undefined reference to `avformat_alloc_output_context2'
ffmpegEncode_sV.c:(.text+0x659): undefined reference to `avformat_alloc_output_context2'
../lib/libsVencode.a(ffmpegEncode_sV.c.o): In function `prepare':
ffmpegEncode_sV.c:(.text+0xa66): undefined reference to `avformat_alloc_output_context2'
ffmpegEncode_sV.c:(.text+0xe4f): undefined reference to `avformat_alloc_output_context2'
collect2: error: ld returned 1 exit status
```

I took the PPA packages as starting point.

**Comment by valgit** [comment]
@ckorn can you start from the "debian" directory in the master branch instead ? the pap is very old.

**Comment by ckorn** [comment]
Thanks, the master branch worked.

Just published it for Ubuntu 15.10:
http://www.getdeb.net/app/slowmoVideo
